api-reference/endpoint/query.mdx

```diff
@@ -0,0 +1,134 @@
+---
+title: "Check Software EOL"
+api: "POST https://eol.xeol.com/api/v1/query"
+description: "This endpoint checks if the provided software identifier(s) are end-of-life"
+---
+
+### Body
+
+<ParamField body="identifiers" type="object[]">
+  list of software identifiers to check for end-of-life status
```

Comment:
we also need to be able to pass in an optional support_level for a software, and think about how we're going to surface the different support levels for a product.

For example, if a have RHEL Extended Support, how do I know which support level number to pass in.